### PR TITLE
Big commit splitting out the blog / weeknotes

### PR DESCRIPTION
**Pull request by idrysdale**

### CODE REVIEW

File: source/weeknotes/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+# Weeknotes
+
+We send out an update on our work each week. Here’s the latest, and there’s
+a <a href="#previous">list of the older ones below</a>.
+
+---
+
+<h2>Latest: <%= blog.articles.first.title %></h2>
```
Comment (paulfurley) [inline source/weeknotes/index.html.md.erb]: Don't think the "Latest:" is that useful

File: source/weeknotes/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<%= blog.articles.first.body %>
```
Comment (paulfurley) [inline source/weeknotes/index.html.md.erb]: It'd be nice if the latest had the date, like the latest blog does

**Comment by paulfurley** [comment]
I'm not convinced that dumping all the blog posts out to one screen is helpful: blog posts are likely to be longer than weeknotes, and it's useful to be able to find the URL for a blogpost to be able to share it

**Comment by paulfurley** [comment]
The `<h1>` on the About page feels a bit underwhelming after the `<h1>` on the `/blog/` page. The red style is kind of awesome

File: source/weeknotes/index.html.md.erb (hunk 1 above)
Comment (paulfurley) [reply source/weeknotes/index.html.md.erb]: Even more radical: I don't think the "Weeknotes" `h1` and following paragraph is useful either. Just go straight into the latest weeknote as an `h1` ( as you've done on `/blog/` )

**Comment by paulfurley** [comment]
the footer is misbehaving on the homepage:

image

and on /weeknotes

File: source/weeknotes/index.html.md.erb (hunk 1 above)
Comment (paulfurley) [reply source/weeknotes/index.html.md.erb]: Suggested in https://github.com/fluidkeys/www.fluidkeys.com/pull/46

Comment (idrysdale) [reply source/weeknotes/index.html.md.erb]: Sounds good! Simpler that way too!

File: source/weeknotes/index.html.md.erb (hunk 2 above)
Comment (idrysdale) [reply source/weeknotes/index.html.md.erb]: Addressed here: https://github.com/fluidkeys/www.fluidkeys.com/pull/51

**Pull request merged**